policy/release/sbom_spdx/sbom_spdx.rego

```diff
@@ -206,3 +206,46 @@ deny contains result if {
 
 	result := lib.result_helper_with_term(rego.metadata.chain(), [purl, qualifier.value], purl)
 }
+
+# METADATA
+# title: Disallowed package attributes
+# description: >-
+#   Confirm the SPDX SBOM contains only packages without disallowed
+#   attributes. By default all attributes are allowed. Use the
+#   "disallowed_attributes" rule data key to provide a list of key-value pairs
+#   that forbid the use of an attribute set to the given value.
+# custom:
+#   short_name: disallowed_package_attributes
+#   failure_msg: Package %s has the attribute %q set%s
+#   solution: Update the image to not use a disallowed package attributes.
+#   collections:
+#   - redhat
+#   - policy_data
+#   effective_on: 2025-02-04T00:00:00Z
+deny contains result if {
+	some s in sbom.spdx_sboms
+	some pkg in s.packages
+
+	some externalref in pkg.externalRefs
+
+	some annotation in pkg.annotations
+	properties := json.unmarshal(annotation.comment)
+	some disallowed in lib.rule_data(sbom.rule_data_attributes_key)
+	properties.name == disallowed.name
+
+	object.get(properties, "value", "") == object.get(disallowed, "value", "")
```

Comment:
If the value key is not present in either then it will produce a violation. Is that intentional? Maybe we're expecting the value key will always be present? (Just trying to understand why it's not just this:)

```suggestion
	properties.value == object.value
```

Reply:
Never mind, I see based on the existing sbom_cyclonedx/sbom_cyclonedx.rego logic.